ocaml/xapi-consts/api_messages.ml

```diff
@@ -365,3 +365,7 @@ let periodic_update_sync_failed = addMessage "PERIODIC_UPDATE_SYNC_FAILED" 3L
 
 let xapi_startup_blocked_as_version_higher_than_coordinator =
   addMessage "XAPI_STARTUP_BLOCKED_AS_VERSION_HIGHER_THAN_COORDINATOR" 2L
+
+let all_vms_in_the_anti_affinity_group_are_running_on_a_single_host =
```

Comment:
May I know why it is named as "all VMs in the group are running on a host"? I think this is not the definition of an anti-affinity breach, I think the definition of breach is "all running VMs in the anti-affinity group resident on one single host", the difference is that, non-running VMs is not counted.

Reply:
We have discussed this in a resolved conversion.

Reply:
I tried to find in the comments discussion thread above but didn't find it so then raised the question here.
Would you please describe it here or share the link to the resolved conversion? Thanks!

Reply:
I think a better name would look like: `all_running_vms_in_the_anti_affinity_grp_resident_on_a_single_host`

Reply:
`Anti_affinity_VM_group_on_single_host`?